pkg/solana/client/test_helpers.go

```diff
@@ -29,17 +29,17 @@ func SetupLocalSolNode(t *testing.T) string {
 func SetupLocalSolNodeWithFlags(t *testing.T, flags ...string) (string, string) {
 	t.Helper()
 
-	port := utils.MustRandomPort(t)
-	portInt, _ := strconv.Atoi(port)
+	port := freeport.GetN(t, 2)
+	portStr := strconv.Itoa(port[0])
 
-	faucetPort := utils.MustRandomPort(t)
-	url := "http://127.0.0.1:" + port
-	wsURL := "ws://127.0.0.1:" + strconv.Itoa(portInt+1) //there is no way to define ws port on Solana validation. It must be +1 from rpc port.
+	faucetPort := freeport.GetOne(t)
+	url := "http://127.0.0.1:" + portStr
+	wsURL := "ws://127.0.0.1:" + strconv.Itoa(port[1]) //there is no way to define ws port on Solana validation. It must be +1 from rpc port.
```

Comment:
Not familiar with the new freeport package but does `GetN` return any 2 free ports? Or are they consecutive? Just asking because of the comment on this line. Think this would have to be `port[0]+1` if they aren't consecutive